### PR TITLE
EL-4202: Column Picker plunker example doesn't compile

### PR DESCRIPTION
**Pull request by JonathanToddMF**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-4202

#### Description of Proposed Changes

Added string | ColumnPickerGroupItem to doc snippet

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-4202-column-picker-doesnt-compile

#### Downstream Build(s)

[ux-aspects-micro-focus](https://jenkins.swinfra.net/job/SEPG/job/caf/view/Developer/job/caf~ux-aspects-micro-focus~EL-4202-column-picker-doesnt-compile~CI/)

### CODE REVIEW

File: docs/app/pages/components/components-sections/tables/column-picker/snippets/app.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Component } from '@angular/core';
-import { ColumnPickerGroup, ColumnPickerValue } from '@ux-aspects/ux-aspects';
+import { ColumnPickerGroup } from '@ux-aspects/ux-aspects';
+import { ColumnPickerGroupItem } from '../../../../../../../../src/components/table/column-picker/interfaces/column-picker-group-item.interface';
```
Comment (donna-marijne) [inline docs/app/pages/components/components-sections/tables/column-picker/snippets/app.ts:3]: Import from @ux-aspects/ux-aspects, applications do not have access to our src directory.

**Review by donna-marijne — CHANGES_REQUESTED** [review]
Please fill in the PR template. Use the user script if you are having trouble with this, it is one click.

**Comment by donna-marijne** [comment]
Please update the PR description.

File: docs/app/pages/components/components-sections/tables/column-picker/snippets/app.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { Component } from '@angular/core';
-import { ColumnPickerGroup, ColumnPickerValue } from '@ux-aspects/ux-aspects';
+import { ColumnPickerGroup } from '@ux-aspects/ux-aspects';
+import { ColumnPickerGroupItem } from '@ux-aspects/ux-aspects/components/table/column-picker/interfaces/column-picker-group-item.interface';
```
Comment (donna-marijne) [inline docs/app/pages/components/components-sections/tables/column-picker/snippets/app.ts:3]: Import this from @ux-aspects/ux-aspects, just like you would in an application (see the line above).

**Review by donna-marijne — CHANGES_REQUESTED** [review]

**Review by donna-marijne — APPROVED** [review]
Verification:

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-4202-column-picker-doesnt-compile/#/components/tables#column-picker - :heavy_check_mark:

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/caf_CI_ux-aspects-micro-focus_EL-4202-column-picker-doesnt-compile/#/ui-components/tables#column-picker - :heavy_check_mark:

**Review by ashley-hunter — APPROVED** [review]
All good, thank you

**Pull request merged**